src/config.ts

```diff
@@ -10,6 +10,7 @@ export type ToolType =
     | 'pytest'
     | 'googlecpp'
     | 'catch2'
+    | 'julia'
```

Comment:
since it's using some specific benchmarking tool for Julia can we rename that to eg. `benchmarkToolsJulia`

Reply:
Considering that this is the only benchmark tool in Julia (and it will be the standard one in a pretty long time), maybe it's safe to just use `julia` here?